test/extended/testdata/kernel/rt-tests-environment.yaml

```diff
@@ -9,7 +9,6 @@ metadata:
     pod-security.kubernetes.io/warn: privileged
   annotations:
     workload.openshift.io/allowed: management
-    openshift.io/node-selector: "node-role.kubernetes.io/worker="
```

Comment:
This was needed when running on GCP but no longer desired as these tests are going to be targeted for SNO on 4.17+